<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogScaleEdit</class>
 <widget class="QDialog" name="DialogScaleEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rButton_ScaleUp">
          <property name="text">
           <string>Up</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rButton_ScaleDown">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spBoxScale">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Scale">
          <property name="text">
           <string>Factor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogScaleEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogScaleEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rButton_ScaleUp</sender>
   <signal>clicked(bool)</signal>
   <receiver>DialogScaleEdit</receiver>
   <slot>scaleUpClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rButton_ScaleDown</sender>
   <signal>clicked(bool)</signal>
   <receiver>DialogScaleEdit</receiver>
   <slot>scaleDownClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBoxScale</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogScaleEdit</receiver>
   <slot>scaleFactorChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scaleUpClicked(bool)</slot>
  <slot>scaleDownClicked(bool)</slot>
  <slot>scaleFactorChanged(int)</slot>
 </slots>
</ui>
